orttraining/orttraining/training_ops/cuda/nn/layer_norm_impl.cu

```diff
@@ -700,14 +392,17 @@ __global__ void cuComputeGradInput(
       }
     }
     // all threads now have the two sums over l
+    // U sum_loss2 = X_mean_difference_over_std_var in cpu kernel
     U fH = (U)n2;
     U term1 = (U(1) / fH) * c_invvar;
     T* k_grad_input = grad_input + i1 * n2;
     if (gamma != NULL) {
       for (int l = thrx; l < n2; l += numx) {
         const U c_loss = static_cast<U>(k_dout[l]);
         U f_grad_input = fH * c_loss * U(gamma[l]);
-        f_grad_input -= sum_loss1;
+        if (!simplified) {
+          f_grad_input -= sum_loss1;
```

Comment:
sum_loss1 can be skipped for computation for SimpliedLayerNorm.... 

But let's do this in other PR, this is an optimization.

Reply:
You're right. I'll do that in another PR.